app/src/org/commcare/dalvik/odk/provider/InstanceProvider.java

```diff
@@ -222,6 +222,7 @@ public Uri insert(Uri uri, ContentValues initialValues) {
 
         SQLiteDatabase db = mDbHelper.getWritableDatabase();
         long rowId = db.insert(INSTANCES_TABLE_NAME, null, values);
+        db.close();
```

Comment:
Trying to get rid of a database error that occurs on first login. Didn't work, but these db.close() statements should probably stay anyways, right?